apps/tailwind-components/app/components/Modal.vue

```diff
@@ -63,56 +64,62 @@ function hide() {
 </script>
 
 <template>
-  <div
-    v-if="visible"
-    role="dialog"
-    :aria-labelledby="title"
-    :aria-modal="true"
-    ref="dialog"
-    class="fixed min-h-lvh w-full top-0 left-0 flex z-30 overscroll-contain"
-  >
-    <div
-      id="backdrop"
-      @click="visible = false"
-      class="w-full h-full absolute left-0 bg-black/60 overscroll-contain"
-      tabindex="-1"
-    />
+  <ClientOnly>
+    <Teleport to="body">
```

Comment:
ah, hadn't seen this feature before. Very useful for this dialogue use case, but generally maybe a bit hard to debug if you don't know where it teleported from. this is needed to get position right I suppose?